addons/website/tests/test_page_manager.py

```diff
@@ -21,6 +21,6 @@ def test_01_page_manager(self):
         self.start_tour(url, 'website_page_manager', login="admin")
```

Comment:
About the commit msg, you can go with
```diff
- Linked to runbot error 55875
+ runbot-55875
```
Same notation as for `task-123` and `opw-123`